lib/screen/scan_qr/scan_qr_page.dart

```diff
@@ -358,6 +375,15 @@ class _ScanQRPageState extends State<ScanQRPage>
     return Stack(
       children: [
         _qrView(),
+        Scaffold(
```

Comment:
looks like we already have `Scaffold`, may add to it instead of creating a new here.